Source/WebKit/UIProcess/BrowsingContextGroup.h

```diff
@@ -25,11 +25,16 @@
 
 #pragma once
 
+#include "WebProcessProxy.h"
```

Comment:
I think this can be a forward declaration instead of including a header from a header.

Reply:
The issue is with `LockdownMode`, which is defined within `WebProcessProxy`.
I'll do a follow up to move it out of the class.